### PR TITLE
Remove unnecessary StringBuilder allocation

### PR DESCRIPTION
**Pull request by SunnyWar**
Replaces unnecessary StringBuilder allocation and replaces it wiht
simple string consctruction.

### CODE REVIEW

**Comment by stephentoub** [comment]
This is in test code. Does it really matter? We can change it, but it's not clear to me why this particular inefficiency in a test is being singled out.

File: src/Common/tests/System/Xml/ModuleCore/cltmconsole.cs [hunk 1]
```diff
@@ -42,9 +42,7 @@ public override void Write(char[] ch)
             //up to be two carriage returns!
             if (ch != null)
             {
-                StringBuilder builder = new StringBuilder(ch.Length);
-                builder.Append(ch);
-                Write(builder.ToString());
+                Write(new String(ch));
```
Comment (stephentoub) [inline src/Common/tests/System/Xml/ModuleCore/cltmconsole.cs]: Nit: String => string

Comment (SunnyWar) [inline src/Common/tests/System/Xml/ModuleCore/cltmconsole.cs]: changed it.

Comment (stephentoub) [inline src/Common/tests/System/Xml/ModuleCore/cltmconsole.cs]: Thanks. What about my other question?

**Comment by SunnyWar** [comment]
I didn't know this is test code....so in the grand scheme of thing it doesn't matter. On the other hand, I've always thought test code should be held to the same standard as production code.

**Comment by stephentoub** [comment]
> On the other hand, I've always thought test code should be held to the same standard as production code.

In some ways, I agree. In others, though, I don't.

We spend a lot of time investing in optimizations for product code, because such changes, even minor, can have significant impact on lots of codebases. That's not the case for test code, where it only impacts this particular codebase and usage. The investment in optimizing test code is thus only valuable if it makes a meaningful and measurable impact on the tests using it. That's not the case here. This change does at least lead to simpler code, which is nice, and we can merge the change, I just want to make sure we're all investing our time and energy in the right place.

Thanks, again.

**Comment by SunnyWar** [comment]
Ya, I should have mentioned that the performance aspect really is irrelevant in this case. I'd argue, though, that this change makes this method much more readable. Easily readable code, no matter where it is, is important.

**Pull request merged**